readthedocs/api/v2/utils.py

```diff
@@ -176,6 +194,22 @@ def run_automation_rules(project, versions_slug):
         rule.run(version)
 
 
+def trigger_external_build(project, version_list):
+    """
+    Trigger Builds for all external versions provided.
+
+    This will mark all the provided external versions as active
+    and trigger build for those versions
+
+    """
+    for version_slug in version_list:
+        version = project.versions(manager=EXTERNAL).get(slug=version_slug)
+        version.active = True
+        version.save()
+
+        trigger_build(project=project, version=version)
```

Comment:
We definitely don't want to trigger all builds for external versions. This will create hundreds of builds on first import. We need a better mechanism for this I think.